examples/OAuth Samples/node_oauth/index.js

```diff
@@ -0,0 +1,92 @@
+// This sample app is intended to demonstrate the OAuth 2.0 flow that is required to call Miro's V2 APIs. 
+// Devs may consider using this Node.js demo as a structural basis for any other preferred language/framework.
+// NOTE: Any comments with "--->" signify part of a significant step in the flow. Comments without "--->" are added for additional reference on code.
+
+// For the full guide on Miro's OAuth 2.0 flow, please see the documentation here:
+// https://beta.developers.miro.com/docs/getting-started-with-oauth 
+
+// ---> #0: 
+// ---> Require dependencies: 'dotenv', 'axios', 'express'
+
+// Require sensitive environment variables (Client ID, Client Secret, Miro Board ID)
+require('dotenv/config');
+
+// Require Axios for handling HTTP requests to Miro /oauth and Miro API endpoints
+const axios = require('axios');
+
+// Require ExpressJS for local server
+const express = require('express');
+const app = express();
+
+// Call Express server
+app.get('/', (req, res) => {
+
+    // ---> #1: 
+    // ---> If an authorization `code` has not already been retrieved from the redirect URL after Miro authorization screen, proceed to #2 (Redirect to Miro authorization URL). Else, proceed to #3 (Requesting access_token/refresh_token pair, using `code`).
+
+    if (req.query.code) {
+
+        // #3: 
+        // ---> Request `access_token` and `refresh_token` pair by making a request to Miro /oauth endpoint. 
+        // ---> Required parameters include `grant_type`, `client_id`, `client_secret`, `code`, and `redirect_uri`.
+        // ---> See full details in Miro documentation here: https://beta.developers.miro.com/docs/getting-started-with-oauth#step-3
+
+        var url = 'https://api.miro.com/v1/oauth/token?grant_type=authorization_code&client_id=' + process.env.clientID + '&client_secret=' + process.env.clientSecret + '&code=' + req.query.code + '&redirect_uri=' + process.env.redirectURL
```

Comment:
Can we be consistent across the board & use `const` or `let` instead of `var`?

Reply:
Great point, thanks!